src/foam/u2/view/UnstyledTableView.js

```diff
@@ -319,7 +311,12 @@ foam.CLASS({
 
     async function initE() {
       var view = this;
-
+      const asyncRes = await this.filterUnpermited(view.of.getAxiomsByClass(foam.core.Property));
+      this.allColumns = ! view.of ? [] : [].concat(
+        asyncRes.map(a => a.name),
+        view.of.getAxiomsByClass(foam.core.Action)
+        .map(a => a.name)
+      );
```

Comment:
`allColumns` are filtered but what about `columns`(those are columns that we use if there are `tableColumns` axiom of class set)?

Reply:
@olhabn yeah, haven't noticed that. thanks, gonna check it